src/api/RoomzSignalingServerClient.js

```diff
@@ -0,0 +1,101 @@
+import io from 'socket.io-client';
+
+// Connect to RSS.
+const rssClientSocket = io("http://localhost:5000", {
+  autoConnect: false,
+  reconnection: true,
+  reconnectionAttempts: 2,
+  timeout: 10000 // timeout for each reconnection attempt
+});
+
+async function askToConnect() {
+  try {
+    await rssClientSocket.connect()
+    return true
+  } catch (e) {
+    console.error("Failed to create socket connection: %s", e)
+    return false
+  }
+}
+
+const events = {
+  JOIN_MEDIA_ROOM: 'JoinMediaRoom',
+  RELAY_ICE_CANDIDATE: 'RelayICECandidate',
+  RELAY_SDP: 'RelaySDP',
+  LEAVE_MEDIA_ROOM: 'LeaveMediaRoom',
+  ADD_PEER: 'AddPeer',
+  INCOMING_ICE_CANDIDATE: 'IncomingICECandidate',
+  INCOMING_SDP: "IncomingSDP",
+  REMOVE_PEER: "RemovePeer",
+}
+
+rssClientSocket.on('disconnect', () => {
+  console.log(':rss: DISCONNECTED from rss');
+})
+
+rssClientSocket.on('reconnect_failed', () => {
+  console.log(':rss: Failed to reconnect. Closing socket.');
+})
+
+function joinMediaRoom(data, cb) {
+  console.log(':rss.joinMediaRoom: Sending request to join media room, data=%o', data)
+  rssClientSocket.emit(events.JOIN_MEDIA_ROOM, data);
+  cb()
```

Comment:
minor: add semicolons to the end of all lines

Reply:
done